rmgpy/molecule/molecule.py

```diff
@@ -1621,4 +1621,41 @@ def toGroup(self):
         
         return group
 
+    def getAromaticSSSR(self):
+        """
+        Returns the smallest set of smallest aromatic rings
```

Comment:
more doctring? (e.g., based on aromatic criterion from RDKit) to show this method is done in a more fundamental way of checking aromaticity instead of just by checking `bond.isBenzene()`